OurUmbraco/Community/GitHub/GitHubService.cs

```diff
@@ -918,6 +919,47 @@ public void UpdateIssues(PerformContext context, Community.Models.Repository rep
                 context.WriteLine("Error:" + ex.Message + " - Stack trace: " + ex.StackTrace);
             }
         }
+        //public void CloseExpiredIssue(GitHubCategorizedIssues expiredIssues)
+        //{
+        //    foreach (var issue in expiredIssues)
+        //    {
+        //        addComment.CommentBody = addComment.CommentBody.Replace("{{issueowner}}", "@" + recentIssue.User.Login);
+        //        var commentsEndPoint = string.Concat(resource, recentIssue.Number, "/comments");
+        //        var request = new RestRequest(commentsEndPoint, Method.POST);
+        //        request.AddHeader("Authorization", "Bearer 6ddf7be25ff5db5522a1a54517f09e41e345726e");
+        //        request.AddHeader("Content-Type", "application/json");
+        //        request.AddParameter("undefined", JsonConvert.SerializeObject(addComment), ParameterType.RequestBody);
+        //        client.UserAgent = UserAgent;
+        //        var result = client.Execute<List<GithubPullRequestModel>>(request);
+
+        //        LogHelper.Info<GitHubService>($"Adding comment to issue {recentIssue.Number}");
+        //    }
+        //}
+
+        public string AddLabelDescriptionAsComment(List<Issue> issues, AddComment addComment, string repository, string label, PerformContext context)
+        {
+            // Initialize the request
+            var username = ConfigurationManager.AppSettings["GitHubUsername"];
+            var password = ConfigurationManager.AppSettings["GitHubPassword"];
+            var client = new RestClient(GitHubApiClient) { Authenticator = new HttpBasicAuthenticator(username, password) };
+            var resource = $"/repos/{RepositoryOwner}/{repository}/issues/";
+            // Make the request to the GitHub API           
+            foreach (var recentIssue in issues)
+            {
+                addComment.CommentBody = addComment.CommentBody.Replace("{{issueowner}}", "@" + recentIssue.User.Login);
+                var commentsEndPoint = string.Concat(resource, recentIssue.Number, "/comments");
+                var request = new RestRequest(commentsEndPoint, Method.POST);
+                request.AddHeader("Authorization", "Bearer 9fb688e28f551ab7278b478145648d09cd6ab662");
+                request.AddHeader("Content-Type", "application/json");
+                request.AddParameter("undefined", JsonConvert.SerializeObject(addComment), ParameterType.RequestBody);
+                client.UserAgent = UserAgent;
+                var result = client.Execute<List<GithubPullRequestModel>>(request);
+
+                context.WriteLine($"Adding comment to issue {recentIssue.Number}");
```

Comment:
As we're listening for more than one repository, we should probably log some information about the repository as well. Eg. start the line with something like `[Umbraco-CMS/1234]`.